wgpu-core/src/command/clear.rs

```diff
@@ -228,6 +220,23 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             });
         }
 
+        let clear_usage = if dst_texture
```

Comment:
could be refactored slightly nicer with a list
```rust
const POSSIBLE_CLEAR_USAGES: &[hal::TextureUses] = [...];
let clear_usage = POSSIBLE_CLEAR_USAGES.iter().find(|&usage| dst_texture.hal_usage.contains(usage)).expect("...");
```

Reply:
not applicable as final version first splits over clear-by-copy and clear-by-pass, making this a distinction between DEPTH_STENCIL_WRITE/COLOR_TARGET only